graph/telemetry/istio/appender/health_annotation.go

```diff
@@ -0,0 +1,75 @@
+package appender
+
+import (
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/graph"
+	"github.com/kiali/kiali/models"
+)
+
+const AnnotationAppenderName = "healthAnnotation"
```

Comment:
"healthAnnotation" is a little confusing to me, I'd prefer a "healthConfig" naming.
Another consideration is that this appender is pretty close to the sidecar one, and perhaps both could be merged together.

But I'd let @jshaughn to comment here.

Reply:
I am goign to replace this by the new syntax "health.kiali.io/rate" , if we add more annotations we can change this to "kiali annotations" to return all annotations like kiali.io

Reply:
The suggestion is to use a "healthConfiguration" appender instead of "healthAnnotations".
When we cover a more generic case we can reconsider, but "healthAnnotation" is a little bit confusing.